python/lsst/ts/m2gui/control_tabs.py

```diff
@@ -53,9 +57,9 @@ class ControlTabs(object):
         Layout.
     """
 
-    def __init__(self, model):
+    def __init__(self, model: Model) -> None:
         # List of the control tables
-        self._tabs = list()
+        self._tabs: typing.List[TabDefault] = list()
```

Comment:
I think you could use `list[TabDefault]`. My impression is that `typing.List` is supposed to be deprecated in favor of `list`.

Reply:
Fixed in the commit: [b8ec396](https://github.com/lsst-ts/ts_m2gui/pull/35/commits/b8ec3962d7117af883c72fd25835442da89caf3d). Thanks!